mvrx/src/main/kotlin/com/airbnb/mvrx/BaseMvRxFragment.kt

```diff
@@ -12,30 +12,22 @@ import androidx.lifecycle.LifecycleOwner
  */
 abstract class BaseMvRxFragment(@LayoutRes contentLayoutId: Int = 0) : Fragment(contentLayoutId), MvRxView {
 
-    private val mvrxViewIdProperty = MvRxViewId()
-    final override val mvrxViewId: String by mvrxViewIdProperty
+    private lateinit var viewDelegate: DelegateMvRxView
+
+    final override val mvrxViewId: String get() = viewDelegate.mvrxViewId
 
     override fun onCreate(savedInstanceState: Bundle?) {
-        mvrxViewIdProperty.restoreFrom(savedInstanceState)
         super.onCreate(savedInstanceState)
+        viewDelegate = delegateMvRxView(
+                subscriptionLifecycleOwner = subscriptionLifecycleOwner,
```

Comment:
I think `subscriptionLifecycleOwner` is not necessary to explicitly define now, as the default is the same?